modules/core/core-app/src/test/java/com/enonic/xp/core/impl/app/ApplicationServiceImplTest.java

```diff
@@ -475,7 +532,9 @@ public void uninstall_local_application()
         this.service.uninstallApplication( application.getKey(), false );
         assertNull( this.service.getInstalledApplication( application.getKey() ) );
 
-        Mockito.verify( this.eventPublisher, Mockito.never() ).publish(
+        verify( this.eventPublisher, Mockito.never() ).publish(
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Mockito.never' due to existing static import 'org.mockito.Mockito.never'](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6220523)